MaxText/layers/linears.py

```diff
@@ -221,7 +221,7 @@ def __call__(self, inputs, decode: bool = False, deterministic: bool = False):
           dtype=self.dtype,
           weight_dtype=self.weight_dtype,
           kernel_init=self.kernel_init,
-          kernel_axes=("embed", "num_activations", "mlp"),
```

Comment:
nit: I would leave these as mlp just b/c the mix of strings and common_types looks strange, but either is fine...

Reply:
lol, np, I am fine with either way. I could change it back without using common_type.